metricbeat/module/elasticsearch/node/data.go

```diff
@@ -93,8 +93,7 @@ func eventsMapping(r mb.ReporterV2, info elasticsearch.Info, content []byte) err
 			continue
 		}
 
-		// Write name here as full name only available as key
-		event.MetricSetFields["name"] = name
+		event.MetricSetFields["id"] = id
```

Comment:
Was this a "bug" before?

Reply:
Yeah, it was. We were actually assigning the node ID to the `name` field in the event. It's easy to get them confused because by default ES will set the node name as the node ID.

Reply:
Good point. Added CHANGELOG entry in breaking changes section in d3750ad7e.